### PR TITLE
Copy static/root to root folder with build #865

### PR DESCRIPTION
**Pull request by harshitkumar31**
**What kind of change does this PR introduce?**
Feature - #865 

**Did you add tests for your changes?**
Yes
**Summary**
Copies the files from static/root directory to the project root.

**Does this PR introduce a breaking change?**
No

**Other information**

Please paste the results of `preact info` here.

cc: @prateekbh

### CODE REVIEW

File: packages/cli/lib/lib/webpack/webpack-client-config.js [hunk 1]
```diff
@@ -109,6 +109,10 @@ async function clientConfig(env) {
 						from: resolve(__dirname, '../../resources/sw-debug.js'),
 						to: 'sw-debug.js',
 					},
+					existsSync(source('static/root')) && {
+						from: resolve(source('static/root')),
+						to: resolve(cwd),
```
Comment (developit) [inline packages/cli/lib/lib/webpack/webpack-client-config.js]: I think this puts them in the repo root. I believe should be `to: '.'` to place them in `output.path`.

File: packages/cli/tests/build.test.js [hunk 2]
```diff
@@ -136,4 +137,11 @@ describe('preact build', () => {
 		let dir = await subject('location-patch');
 		expect(() => build(dir)).not.toThrow();
 	});
+
+	it('should copy resources from static/root to root directory', async () => {
+		let dir = await subject('static-root');
+		await build(dir);
+		let file = join(dir, '.htaccess');
```
Comment (developit) [inline packages/cli/tests/build.test.js]: Static root files should still be placed into the build output directory:

```suggestion
		let file = join(dir, 'build', '.htaccess');
```

File: packages/cli/lib/lib/webpack/webpack-client-config.js [hunk 3]
```diff
@@ -109,6 +109,10 @@ async function clientConfig(env) {
 						from: resolve(__dirname, '../../resources/sw-debug.js'),
 						to: 'sw-debug.js',
 					},
+					existsSync(source('static/root')) && {
```
Comment (prateekbh) [inline packages/cli/lib/lib/webpack/webpack-client-config.js]: it does not need to be in `static/root` it was an option to pic a name. just pick any one.

**Comment by prateekbh** [comment]
Implementation LGTM. Can someone take a look if `root` is good name choice for the folder

// @ForsakenHarmony

**Comment by ForsakenHarmony** [comment]
Yeah, probably not

myb `static` or `public`

**Pull request merged**